pkg/manager/manager.go

```diff
@@ -115,12 +121,17 @@ func (k *KubeMacPoolManager) Run(rangeStart, rangeEnd net.HardwareAddr) error {
 		}
 
 		// create a owner ref on the mutating webhook
-		// this way when we remove the statefulset of the manager the webhook will be also removed from the cluster
+		// this way when we remove the deployment of the manager the webhook will be also removed from the cluster
 		err = webhook.CreateOwnerRefForMutatingWebhook(k.clientset, k.podNamespace)
```

Comment:
This function is namespaced, don't we want to keep the new owner ref function in the same place?

Reply:
done